changes/5221.fixed

```diff
@@ -0,0 +1 @@
+Fixed the return url from adding/assigning a contact/team from ObjectDetailView to redirect to the contacts tab instead of the main tab.
```

Comment:
```suggestion
Fixed the return URL from adding/assigning a contact/team from ObjectDetailView to redirect to the contacts tab instead of the main tab.
```